GoogleMulticastAppDelegate.podspec

```diff
@@ -0,0 +1,72 @@
+Pod::Spec.new do |s|
+  # TODO: Is `GoogleMulticastAppDelegate` name fine?
```

Comment:
Is this useful for non-Firebase libraries? If no, how about FirebaseMulticastAppDelegate. If yes, it should be GoogleUtilitiesMulticastAppDelegate to avoid expanding our `Google` namespace.

Reply:
Oh, I think it's needed for GoogleAppMeasurement and AdMob, so probably needs to stay out of the Firebase repo.

Reply:
And if it's used by GoogleAppMeasurement, I guess we should keep it Google prefixed?

Reply:
Yes. Even though it's long we should keep it `GoogleUtilities` prefixed since we have approval for that, but not generic `Google` prefix.